src/pages/BrowseSchemes.svelte

```diff
@@ -149,6 +150,7 @@
             <CombinedAuthoritiesLayerControl />
             <LocalAuthorityDistrictsLayerControl />
           </CheckboxGroup>
+          <CensusOutputAreaLayerControl />
```

Comment:
If I use the regular checkbox group above, the legends on different lines blow up:
![Screenshot from 2023-08-02 13-04-48](https://github.com/acteng/atip/assets/1664407/a0636e75-fbc2-4bd1-a3b3-c334d92fff0b)
Help welcome.